### PR TITLE
BF: closes 1484 for unordered map

### PR DESCRIPTION
**Pull request by MichaelWS**
Closes issue 1484

### CODE REVIEW

File: tests/run/cpp_stl_cpp11.pyx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# tag: cpp, werror, cpp11
+# distutils: extra_compile_args=-std=c++0x
+
+import sys
+from libcpp.unordered_map cimport unordered_map
+from libcpp.pair cimport pair
+
+py_set = set
+py_xrange = xrange
+py_unicode = unicode
```
Comment (scoder) [inline tests/run/cpp_stl_cpp11.pyx]: These seem unused.

**Comment by scoder** [comment]
Otherwise looks good to me now.

**Comment by scoder** [comment]
Thanks!

**Comment by MichaelWS** [comment]
thank you for the help

**Pull request merged**